fix(ResponsiveWebapp): Do not reformat URL after auth0 login.

binh-dam-ibigroup opened the pull request:
Fixes #333 by not reformatting URLs in `/?code=xxxx...`.

landonreed reviewed (approved):
Looks good! Thanks for diagnosing.

Only non-blocking comments are that:
1. I'm not a huge fan of the extra whitespace, and especially the commented line with no text.
2. It wouldn't be a huge lift to also check that search contains `&state=`, which I think might give us more assurance that we're not interfering with Auth0 login -- but I don't feel too strongly about this.

binh-dam-ibigroup commented:
> Looks good! Thanks for diagnosing.
> 
> Only non-blocking comments are that:
> 
> 1. I'm not a huge fan of the extra whitespace, and especially the commented line with no text.
> 2. It wouldn't be a huge lift to also check that search contains `&state=`, which I think might give us more assurance that we're not interfering with Auth0 login -- but I don't feel too strongly about this.

1. I removed the blank line in the comment, but would rather visually separate chunks of comments/logic in the code.
2. I added checking presence of the `state` search param, now using `qs` to parse the search query.
